### PR TITLE
fix: title 不使用 Markdown 语法

### PR DESCRIPTION
**Pull request by crazyair**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: .github/workflows/issue-open-check.yml [hunk 1]
```diff
@@ -99,7 +99,7 @@ jobs:
           DING_TALK_TOKEN: |
             ${{ secrets.DINGDING_BOT_TOKEN }}
             ${{ secrets.DINGDING_BOT_COLLABORATOR_TOKEN }}
-          notify_title: '🔥 @${{ github.event.issue.user.login }} 创建了 issue：[${{ github.event.issue.title }}](${{ github.event.issue.html_url }})'
+          notify_title: '🔥 @${{ github.event.issue.user.login }} 创建了 issue：${{ github.event.issue.title }} ${{ github.event.issue.html_url }}'
```
Comment (Wxh16144) [inline .github/workflows/issue-open-check.yml:102]: 这个改动的话， 推送消息的 title 会太长了吧？

Comment (Wxh16144) [reply .github/workflows/issue-open-check.yml:102]: 我有强迫症，所以当初我提交 pr 就留了 id， 然后自己用 alfred 搞了一下

Comment (crazyair) [reply .github/workflows/issue-open-check.yml:102]: 这个 title 字段不是聊天窗口消息展示用的

Comment (crazyair) [reply .github/workflows/issue-open-check.yml:102]: 这个不是在消息界面展示的，notify_body 才是

Comment (Wxh16144) [reply .github/workflows/issue-open-check.yml:102]: image

Comment (crazyair) [reply .github/workflows/issue-open-check.yml:102]: title 不支持 Markdown 不支持 html

Comment (crazyair) [reply .github/workflows/issue-open-check.yml:102]: image

**Review by zombieJ — APPROVED** [review]

**Pull request merged**